test/StreamingZipReader.UnitTests/StreamingZipReaderTests.cs

```diff
@@ -24,4 +24,19 @@ public async Task BasicTest()
         await reader.MoveToNextEntryAsync(true, CancellationToken.None);
         Assert.Equal((uint) 1024, reader.CurrentEntry.Length);
     }
+
+    [Fact]
+    public async Task Zip64Test()
+    {
+        using var stream = File.OpenRead("test.zip");

```

Comment:
This certainly works! If you add a .resx file to the test project and add the test file to it instead of using CopyToOutputDirectory, this could be something like `new MemoryStream(TestResources.BasicZip64)`.

Reply:
I'm not aware of resx, isn't that a Visual Studio thing?

Reply:
It's a .NET thing, but VS has a UI for it and it's harder to use without a UI.